### DISCUSSION TITLE
Unpack failure for l4t-nvidia-glheaders-32.5.0-r0

### DISCUSSION THREAD

**Discussion by balister** [OE4T/meta-tegra#575] [Q&A] [answered]
I'm new to meta-tegra. Building for nano 2GB machine I see this. I think it downloads the deb OK, but something goes bad in unpack:

ERROR: l4t-nvidia-glheaders-32.5.0-r0 do_unpack: Unpack failure for URL: 'https:
//repo.download.nvidia.com/jetson/t186/pool/main/n/nvidia-l4t-graphics-demos/nvi
dia-l4t-graphics-demos_32.5.0-20210115151051_arm64.deb;subdir=l4t-graphics-demos
'. Unpack command PATH="/home/balister/customers/CROPCIRCLES/nvidia/nano-work/bu
ild/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-s
ysroot-native/usr/bin/tar-l4t-workaround-native:/home/balister/customers/CROPCIR
CLES/nvidia/nano-work/openembedded-core/scripts:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-sysroot-native/usr/bin/aarch64-oe-linux:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-sysroot/usr/bin/crossscripts:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-sysroot-native/usr/sbin:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-sysroot-native/usr/bin:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-sysroot-native/sbin:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/recipe-sysroot-native/bin:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/bitbake/bin:/home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/hosttools" ar x /home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/downloads/nvidia-l4t-graphics-demos_32.5.0-20210115151051_arm64.deb data.tar.zst && tar --no-same-owner -xpf data.tar.zst && rm data.tar.zst failed with return value 2
ERROR: Logfile of failure stored in: /home/balister/customers/CROPCIRCLES/nvidia/nano-work/build/tmp-glibc/work/armv8a_tegra-oe-linux/l4t-nvidia-glheaders/32.5.0-r0/temp/log.do_unpack.4568

**Comment by kekiefer** [comment, marked as answer]
Assuming this is on `dunfell-l4t-r32.5.0`? There were fixes put a few days ago (#559) so confirm that you are up-to-date first. Same fixes are present on master.

**Reply by madisongh** [reply]
It turns out I didn't get that fix quite right after all.  I've just merged another change that should finally take care of the problem.

**Reply by ichergui** [reply]
Thanks a lot @madisongh

**Reply by balister** [reply]
Updated my layer set that follows master and things look much better. Thanks!

**Reply by ichergui** [reply]
@madisongh @kekiefer 

Could we close this or mark it as done ?
I think it have been resolved almost one month ago.
